staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -82,10 +82,21 @@ func validator(s *schema.Structural, isResourceRoot bool, declType *model.DeclTy
 	if len(s.Properties) > 0 {
 		propertiesValidators = make(map[string]Validator, len(s.Properties))
 		for k, prop := range s.Properties {
```

Comment:
should we declare a local var to the loop instead of taking the address of `&prop` in a couple places below?

Reply:
Thanks for catching.